specification/ml/_types/Datafeed.ts

```diff
@@ -197,6 +201,12 @@ export class DatafeedTimingStats {
   average_search_time_per_bucket_ms?: DurationValue<UnitFloatMillis>
 }
 
+export class ExponentialAverageCalculationContext {
+  incremental_metric_value_ms: double
```

Comment:
Maybe `DurationValue<UnitFloatMillis>` here as well like for `previous_exponential_average_ms`?